### PR TITLE
Beam utils doc update

### PR DESCRIPTION
**Pull request by beczkowb**

### CODE REVIEW

File: docs/technologies.md [hunk 1]
```diff
@@ -83,6 +87,51 @@ google_cloud_options.staging_location = f"gs://{staging_location}"
...
+
+wordcount_workflow = bigflow.Workflow(
+    workflow_id="wordcount",
+    log_config={
+        'gcp_project_id': workflow_config['gcp_project_id'],
+        'log_level': 'INFO',
+    },
+    definition=[BeamJob(
+        id='wordcount_job',
+        driver_callable=wordcount_driver,
```
Comment (iwko) [inline docs/technologies.md:120]: Why driver_callable? Why not job_callable?

Comment (beczkowb) [reply docs/technologies.md:120]: To make it a bit similar to the dataproc job.

Comment (iwko) [reply docs/technologies.md:120]: What we will gain thanks to this similarity?

Comment (beczkowb) [reply docs/technologies.md:120]: consistent experience - if you know what is the driver in a dataproc job, then you might easier figure out what is a driver in a dataflow job

Comment (iwko) [reply docs/technologies.md:120]: I am not sure if this is a good way. We introduce a new concept that is not commonly connected with a dataflow.

Comment (beczkowb) [reply docs/technologies.md:120]: well, if you call this parameter `job_callable` it's confusing than helpful - job inside a job? should it implement the `execute` method? etc.
@anjensan what do you think?

Comment (beczkowb) [reply docs/technologies.md:120]: #113

**Review by iwko — APPROVED** [review]

**Pull request merged**